tests/microbenchmarks.py

```diff
@@ -203,10 +212,11 @@ def test_dict_set(test_case):
             dict["meow"] = 4
             dict["cockadoodledoo"] = 5
             dict["hiss"] = 6
-    """, timed=True)
+    """), timed=True)
+
 
 def test_class_init(test_case):
-    print("Running" , "test_class_init")
+    print("Running", "test_class_init")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils